src/modules/crewTypes.ts

```diff
@@ -46,15 +46,16 @@ export interface Vacancy {
     unit: number | null;
     grade: number | null;
 
+    closedAt: Date | null;
```

Comment:
U don't need to use null type. This type exists on db level only. In UI it must be undefined only. Do not push developers write smth like `<Link href={null} />`